adal/src/main/java/com/microsoft/aad/adal/DeserializationAuthenticationException.java

```diff
@@ -28,15 +28,15 @@
  */
 class DeserializationAuthenticationException extends AuthenticationException {
     static final long serialVersionUID = 1;
-    final static ADALError defaultErrorCode = ADALError.INCOMPATIBLE_BLOB_VERSION;
+    static final ADALError DEFAULT_ERROR_CODE = ADALError.INCOMPATIBLE_BLOB_VERSION;
```

Comment:
>DEFAULT_ERROR_CODE [](start = 27, length = 18)

I would just delete this line and use ADALError.INCOMPATIBLE_BLOB_VERSION in its place